tests/kamereon/test_kamereon_vehicle_action.py

```diff
@@ -31,3 +33,116 @@ def test_vehicle_action_response_attributes() -> None:
         "action": "start",
         "targetTemperature": 21.0,
     }
+
+
+def test_charge_mode_for_json() -> None:
+    """Test for updating charge settings."""
```

Comment:
I think there is a small typo here in the duplicated code.

`test_charge_mode_for_json` and `test_hvac_schedule_for_json`  should both have:
- read model
- export and check original json
- update model
- export and check new json

But it seems `test_charge_mode_for_json` has the second step twice, and `test_hvac_schedule_for_json` doesn't have steps 3 and 4

Reply:
I didn’t implement updating for the developer-facing side of the API yet, as i originally planned to add it later together with the CLI. Will do now, and then of course add it to the tests …

Reply:
I've just noticed that the duplicate code on the original data is on my code :D sorry about that.

For the update, just add a simple "manual" update. For example:
```
   assert vehicle_data.schedules[2].monday = None
```
and then make sure it has disappeared on the second call.

Reply:
> I've just noticed that the duplicate code on the original data is on my code :D sorry about that.

i took the liberty of removing in #99 since i had to touch the part anyways. If we could merge #99 first, rebasing this one would be easier, i think.

Reply:
But I think you’re right … for generalizing the update logic, I’d like to move `update_schedule` into the respective schedule-classes …. And I’d like to get the base part for the hvac schedule in finally. So … as you suggested, I’ll add a few manual updates, and do the general logic later, probably together with the CLI.